FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -1033,6 +1036,355 @@ paths:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      security:
+      - OAuth2: [fraihmwork.component.admin]
+
+  /monitor/v1/relationship/instance:
+    get:
+      tags:
+      - Relationships
+      - Monitor
+      summary: Retrieve all of the relationship instances active in FRAIHMWORK, filtered by criteria
+      operationId: findRelationshipInstances
+      parameters:
+      - name: type
+        in: query
+        description: Filter results by the type of relationship. The string should match an existing RelationshipDefinition. Can leave null to not filter on this.
+        schema:
+          type: string
+      - name: source
+        in: query
+        description: Filter results to only return relationships with a source UUID equal to this value. Can leave null to not filter on this.
+        schema:
+          type: string
+          format: uuid
+      - name: target
+        in: query
+        description: Filter results to only return relationships with a target UUID equal to this value. Can leave null to not filter on this.
+        schema:
+          type: string
+          format: uuid
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/RelationshipInstance'
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        403:
+          description: Forbidden - client is not authorized to read components
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      deprecated: false
+      security:
+      - OAuth2: [fraihmwork.component.read, fraihmwork.component.write, fraihmwork.component.admin]
+    post:
+      tags:
+      - Relationships
+      - Monitor
+      summary: Add a new relationship instance
+      description: Relationship instances connect two FRAIHMWORK resources together in a pattern established by existing relationship definitions to describe how they are related to an end user, as well as prescribe some automated actions. This endpoint allows for two resources have a relationship defined between them.
+      operationId: postRelationshipInstance
+      requestBody:
+        description: JSON description of the new relationship instance
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RelationshipInstance'
+      responses:
+        201:
+          description: Successfully posted new component information
```

Comment:
```suggestion
          description: Successfully posted new component instance
```

Reply:
suggest **component relationship instance** instead to make it more clear

Reply:
We are all wrong again! Specifying 'relationship instances' instead